src/pages/en/developers/api-registration/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Developer portal and registration
+order: -12
+---
+
+## Digitransit developer portal
+Starting from 3.4.2023, the use of the Digitransit production APIs ([api.digitransit.fi](https://api.digitransit.fi), [cdn.digitransit.fi](https://cdn.digitransit.fi) and [digitransit-prod-cdn-origin.azureedge.net](https://digitransit-prod-cdn-origin.azureedge.net)) will require registration and use of API keys. Registration can be done at the [Digitransit developer portal](https://portal-api.digitransit.fi/). After the registration, it's possible to create subscriptions to a product and the API keys related to the subscriptions will also be available at the developer portal.
+
+## Transition period
+Developers will have until 3.4.2023 to register at [the developer portal](https://portal-api.digitransit.fi/) and to start using API keys in the requests. After that, we will prevent the anonymous use of our production APIs.
```

Comment:
Developers will have time until 3.4.2023 to register until 3.4.2023 at [the developer portal](https://portal-api.digitransit.fi/) and to start using API keys in requests. After that, we will prevent anonymous use of production APIs.